src/backend/kvm/thread.rs

```diff
@@ -117,7 +120,42 @@ impl<P: KeepPersonality> Thread<P> {
                 };
                 Ok(None)
             }
+            item::Enarxcall {
+                num: item::enarxcall::Number::Park,
+                argv: [val, timeout, ..],
+                ret,
+                ..
+            } => {
+                let timeout = if *timeout != sallyport::NULL {
+                    Some(unsafe {
+                        // Safety: `deref_aligned` gives us a pointer to an aligned `timespec` struct.
+                        // We also know, that the resulting pointer is inside the allocated sallyport block, where `data`
+                        // is a subslice of.
+                        (*deref_aligned::<MaybeUninit<timespec>>(data, *timeout, 1)
+                            .map_err(io::Error::from_raw_os_error)
+                            .context("sgx_enarxcall deref")?)
```

Comment:
This seems to be neither useful, nor in the right place (this is KVM, not SGX)

Why not write an actual sentence looking something like:
> "failed to dereference X in syscall Y"
?

Reply:
fixed